packages/datadog-plugin-express/test/index.spec.js

```diff
@@ -986,6 +985,207 @@ describe('Plugin', () => {
           })
         })
       })
+
+      describe('with configuration for middleware disabled', () => {
+        before(() => {
+          return agent.load(plugin, 'express', {
+            middleware: false
+          })
+        })
+
+        after(() => {
+          return agent.close()
+        })
+
+        beforeEach(() => {
+          express = require(`../../../versions/express@${version}`).get()
+        })
+
+        it('should not activate a scope per middleware', done => {
```

Comment:
This test should explicitly try to change the scope in the previous middleware to ensure the next one properly reactivates the scope of the request and that it's not just propagated because of the execution context. This is to make sure you get the right scope when the previous middleware lost it somehow.

I assume this also applies to other plugin tests.

Reply:
makes sense, updated these tests(along with those in connect and koa) to more explicitly test that.